client/src/components/InputTextField/InputTextField.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./InputTextField.css";
+
+const InputTextField = props => {
+  let { type, name, placeholder, onChange } = props;
+  let classList = `input-field-container-${type}`;
+  InputTextField.defaultProps = {
+    type: "default"
+  };
```

Comment:
Can you move this to outside the component definition?

Read this for more details: https://blog.logrocket.com/a-complete-guide-to-default-props-in-react-984ea8e6972d/

Reply:
Added default value to destructor set variation to "default" by default.